src/components/AuthLoggedIn.tsx

```diff
@@ -0,0 +1,48 @@
+import { Avatar, Button, useMediaQuery } from '@mui/material';
+import React from 'react';
+import { useAuthState, useSignOut } from 'react-firebase-hooks/auth';
+import { authState } from '~configs/firebase';
+import { useAlert } from '~utils/userHooks';
+import { BurgerMenu } from './BurgerMenu';
+import { Stack } from '@mui/system';
+import useTranslation from '~utils/localization';
+
+export const AuthLoggedIn: React.FC = () => {
+  const localization = useTranslation();
+
+  const isSmallScreen = useMediaQuery('(max-width: 680px)');
+  const [user] = useAuthState(authState);
+  const [signOut, loading, error] = useSignOut(authState);
+  const showMsg = useAlert();
+
+  const signOutHandler = async () => {
+    const result = await signOut();
+    if (error) {
+      showMsg({ type: 'error', content: error.message });
+    }
+    if (result) {
+      showMsg({ type: 'info', content: localization.auth.logoutSuccess });
+    }
+  };
+
+  const menuLink = [
+    {
+      text: localization.auth.logout,
+      onClick: () => signOutHandler(),
+    },
+  ];
+
+  return (
+    <Stack direction="row" alignItems="center" gap={2}>
+      <Avatar sx={{ textTransform: 'capitalize', background: '#7c4b4b' }}>
```

Comment:
It is better to put color hex code into variable. The same note for other files.